__tests__/app.test.js

```diff
@@ -156,5 +156,61 @@ describe("/api/articles", () => {
         const { message } = body;
         expect(message).toBe("Endpoint not found");
       });
+  });
+});
+
+describe("/api/articles/:article_id/comments", () => {
+  test("GET 200: Should return an array of comments for the given article ID with the required properties", () => {
+    return request(app)
+      .get("/api/articles/5/comments")
+      .expect(200)
+      .then(({ body }) => {
+        const { comments } = body;
+        comments.forEach((comment) => {
+          expect(comment.article_id).toBe(5);
+          expect(typeof comment.comment_id).toBe("number");
+          expect(typeof comment.votes).toBe("number");
+          expect(typeof comment.created_at).toBe("string");
+          expect(typeof comment.author).toBe("string");
+          expect(typeof comment.body).toBe("string");
+          expect(typeof comment.article_id).toBe("number");
```

Comment:
you don't need to check the type of the article_id, as you've already implicitly done it on line 170 😸